src/main/java/com/glovoapp/ownership/OwnershipAnnotationDefinition.java

```diff
@@ -53,7 +56,11 @@
     static <A extends Annotation> OwnershipAnnotationDefinition define(@NonNull final Class<A> annotationClass,
                                                                        @NonNull final Function<A, ?> ownerGetter,
                                                                        @NonNull final Function<A, Map<String, ?>> metaDataGetter) {
-        ParentPackageAnnotationScanner<A> parentPackageAnnotationScanner = new ParentPackageAnnotationScanner(annotationClass);
+        AnnotationScanner<A> parentPackageAnnotationScanner =
+                new CachedParentPackageAnnotationScanner<A>(
+                        new ParentPackageAnnotationScanner(annotationClass)
```

Comment:
[IDE] Please pay more attention to [your IDE warnings](https://www.jetbrains.com/help/idea/code-inspection.html). This code shows:

> Raw use of parameterized class 'ParentPackageAnnotationScanner' 

```suggestion
                new CachedParentPackageAnnotationScanner<>(
                        new ParentPackageAnnotationScanner<>(annotationClass)
```

ᴛʜɪs ᴀᴄᴛɪᴏɴ ᴡᴀs ᴘᴀʀᴛɪᴀʟʟʏ ᴀᴜᴛᴏᴍᴀᴛᴇᴅ ʙʏ [GɪᴛHᴜʙ's ʀᴇsᴘᴏɴsᴇ ᴛᴇᴍᴘʟᴀᴛᴇs](https://github.com/settings/replies)